src/config.json

```diff
@@ -7,5 +7,5 @@
   "asn": "enter the ASNs of interest here",
   "peerIp": "enter peer IP",
   "connector": "enter connector or connectors of interests",
-  "windowSize": 360
+  "windowSize": "360"
```

Comment:
Just curious. This feels like it should be an int? Considering in the message_processor you end up converting it back to an int.

Reply:
https://github.com/tyakovenko/Fastly-MQP23/pull/8/commits/9b5064691af7673081316cf9cc185b9139c659c6#diff-d40c9dcb832416aceb1048d20f6daf87b81bdaf9595645af106362b6ea5208c2R14

Reply:
agreed, ought to be an int